traffic_ops_ort/t3c/torequest/torequest.go

```diff
@@ -1281,83 +1281,103 @@ func (r *TrafficOpsReq) ProcessPackages() error {
 		// check if the full package version is installed
 		fullPackage := pkgs[ii].Name + "-" + pkgs[ii].Version
 
-		if instpkg == fullPackage {
-			log.Infof("%s Currently installed and not marked for removal\n", reqpkg)
-			r.pkgs[fullPackage] = true
-			continue
-		} else if instpkg != "" { // the installed package needs upgrading.
-			log.Infof("%s Currently installed and marked for removal\n", instpkg)
-			uninstall = append(uninstall, instpkg)
-			// the required package needs installing.
-			log.Infof("%s is Not installed and is marked for installation.\n", fullPackage)
-			install = append(install, fullPackage)
-			// get a list of packages that depend on this one and mark dependencies
-			// for deletion.
-			arr, err = util.PackageInfo("pkg-requires", instpkg)
-			if err != nil {
-				return err
-			}
-			if len(arr) > 0 {
-				for jj := range arr {
-					log.Infof("%s is Currently installed and depends on %s and needs to be removed.", arr[jj], instpkg)
-					uninstall = append(uninstall, arr[jj])
+		if r.Cfg.RunMode == config.BadAss {
+			if instpkg == fullPackage {
+				log.Infof("%s Currently installed and not marked for removal\n", reqpkg)
+				r.pkgs[fullPackage] = true
+				continue
+			} else if instpkg != "" { // the installed package needs upgrading.
+				log.Infof("%s Currently installed and marked for removal\n", instpkg)
+				uninstall = append(uninstall, instpkg)
+				// the required package needs installing.
+				log.Infof("%s is Not installed and is marked for installation.\n", fullPackage)
+				install = append(install, fullPackage)
+				// get a list of packages that depend on this one and mark dependencies
+				// for deletion.
+				arr, err = util.PackageInfo("pkg-requires", instpkg)
+				if err != nil {
+					return err
 				}
+				if len(arr) > 0 {
+					for jj := range arr {
+						log.Infof("%s is Currently installed and depends on %s and needs to be removed.", arr[jj], instpkg)
+						uninstall = append(uninstall, arr[jj])
+					}
+				}
+			} else {
+				// the required package needs installing.
+				log.Infof("%s is Not installed and is marked for installation.\n", fullPackage)
+				log.Errorf("%s is Not installed and is marked for installation.\n", fullPackage)
+				install = append(install, fullPackage)
+			}
+		} else if r.Cfg.RunMode == config.SyncDS {
+			// Only check if packages exist and complain if they are wrong.
+			if instpkg == fullPackage {
+				log.Infof("%s Currently installed.\n", reqpkg)
+				r.pkgs[fullPackage] = true
+				continue
+			} else if instpkg != "" { // the installed package needs upgrading.
+				log.Infof("%s Wrong version currently installed.\n", instpkg)
+				log.Errorf("%s Wrong version currently installed.\n", instpkg)
```

Comment:
Nitpick: if this is an Error, doesn't need to also be an Info.

Reply:
i'll remove those.